### DISCUSSION TITLE
FritzBox 7430 Power LED leuchtet, kein Zugriff auf Webinterface

### DISCUSSION THREAD

**Discussion by lehmanju** [Freetz-NG/freetz-ng#640] [Allgemein] [answered]
Hi, ich hab ein kleines Problem Freetz-NG auf meiner Fritzbox 7430 zum Laufen zu bekommen.

Ich hab die 7.29 Firmware drauf und kompiliere alles auf ArchLinux.

Also zum einen hab ich beim Kompilieren schon das Problem, dass der 1. Step failed:
```
source firmware file: dl/fw/FRITZ.Box_7430-07.29.image
checking signature: failed
OpenSSL: 40273107847F0000:error:02000068:rsa routines:ossl_rsa_verify:bad signature:crypto/rsa/rsa_sign.c:430:
40273107847F0000:error:1C880004:Provider routines:rsa_verify:RSA lib:providers/implementations/signature/rsa_sig.c:774:
Verification failure
ERROR: image could not be verified with public key
```

Dann hab ich das Überprüfen der Signatur mittels menuconfig ausgeschaltet und das Image baut, das Ganze via update script geflasht, aber dann bootet die Box nicht mehr. Alle LEDs sind aus. Man kann lediglich die Stromzufuhr kappen, dann wird die Box auf die vorherige Firmware (7.28) zurückgesetzt. Bei der Konfiguration hab ich alle Packages entfernt und sonst die Standardeinstellungen benutzt. Irgendwelche Ideen?

EDIT:
Mit dem Docker Image funktioniert die Signaturüberprüfung. Das entstandene Image scheint irgendwie anders zu sein, nach dem Flashen leuchtet die Power LED dauerhaft. Allerdings ist ein Zugriff auf das Webinterface weiter nicht möglich. Nach einem Power Reset ist wieder die alte Firmware installiert.

**Comment by lehmanju** [comment]
Die Signaturprüfung läuft und zumindest die Power LED leuchtet jetzt, allerdings scheint das Image trotzdem nicht zu booten.

**Comment by fda77** [comment]
Wenn die Signaturprüfung fehlschlägt ist das Image kaputt. Dass das nachher nicht startet kann schon vorkommen...

**Comment by lehmanju** [comment, marked as answer]
Hat sich erledigt, habe mir einfach eine 7490 zugelegt. Da läuft alles bestens.
